js/utils/urlUtils.ts

```diff
@@ -13,6 +13,8 @@ export const isLocal = () => {
   return url.indexOf('localhost') > -1;
 };
 
+export const isLocalOrLabs = () => isLocal() || isProd();
```

Comment:
Denne skal vel være isLocal og isLabs, ikke isProd

Reply:
zomg

Reply:
skylder på autocomplete :P

Reply:
men bra spotta, ellers hadde det ikke blitt noe tess :D

Reply:
Haha, been there done that :P